esphome/components/esp32_can/esp32_can.h

```diff
@@ -21,6 +23,8 @@ class ESP32Can : public canbus::Canbus {
 
   int rx_{-1};
   int tx_{-1};
+  optional<int> tx_queue_len_{};
+  optional<int> rx_queue_len_{};
```

Comment:
Please use the correct types
```suggestion
  optional<uint32_t> tx_queue_len_{};
  optional<uint32_t> rx_queue_len_{};
```